packages/transport/src/modules/accounts/backbone/IdentityDeletionProcessClient.ts

```diff
@@ -1,12 +1,13 @@
 import { RESTClientAuthenticate, RESTClientLogDirective } from "../../../core";
 import { ClientResult } from "../../../core/backbone/ClientResult";
 import { BackboneIdentityDeletionProcess } from "./BackboneIdentityDeletionProcess";
+import { BackbonePostIdentityDeletionProcessRequest } from "./BackbonePostIdentityDeletionProcess";
 
 export class IdentityDeletionProcessClient extends RESTClientAuthenticate {
     protected override _logDirective = RESTClientLogDirective.LogResponse;
 
-    public async initiateIdentityDeletionProcess(): Promise<ClientResult<BackboneIdentityDeletionProcess>> {
-        return await this.post<BackboneIdentityDeletionProcess>("/api/v1/Identities/Self/DeletionProcesses", {});
+    public async initiateIdentityDeletionProcess(value: BackbonePostIdentityDeletionProcessRequest): Promise<ClientResult<BackboneIdentityDeletionProcess>> {
+        return await this.post<BackboneIdentityDeletionProcess>("/api/v1/Identities/Self/DeletionProcesses", value, {});
```

Comment:
@jkoenig134 Should the third input value (`{}`) be removed as you suggested in the other PR (https://github.com/nmshd/runtime/pull/358#discussion_r1875494942)?

Reply:
yes! We don't want to pass a config here.

Reply:
Already done, just not pushed yet. :)